### PR TITLE
WindowsUpdate PowerShell reference update

### PR DESCRIPTION
**Pull request by TimShererWithAquent**
[1871096](https://dev.azure.com/mseng/TechnicalContent/_workitems/edit/1871096)

Change in syntax reflects deprecated param.

### CODE REVIEW

**Comment by TimShererWithAquent** [comment]
@tiburd These changes are all from PlatyPS, so this PR is ready for final reviews.

@PatAltimore for awareness.

File: docset/winserver2022-ps/windowsupdate/Get-WindowsUpdateLog.md
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/windowsupdate/Get-WindowsUpdateLog.md:199]:
```suggestion
The **SymbolServer** parameter is deprecated for Windows 1709 (OS Build 16299) and later.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review - @TimShererWithAquent, kindly commit the suggested change (we refer to software versions as "later," not "higher"). Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by tiburd** [comment]
@TimShererWithAquent - we're still waiting for the review from @tseward . 

(@JohanFreelancer9 did a copy edit pass, not a technical review.)

**Review by ghost — APPROVED** [review]

**Comment by TimShererWithAquent** [comment]
@tiburd I see @tseward approved. Thanks -

**Pull request merged**
